kong/dao/migrations/cassandra.lua

```diff
@@ -487,6 +487,7 @@ return {
   {
     name = "2017-05-19-173100_remove_nodes_table",
     up = [[
+      DROP INDEX IF EXISTS nodes_cluster_listening_address_idx_index;
```

Comment:
In my local C* 3.9 cluster, this index is named  `nodes_cluster_listening_address_idx`. Oversight?

Reply:
Thanks for catching this! Yes, I mixed up the name for the materialized view and for the actual index.